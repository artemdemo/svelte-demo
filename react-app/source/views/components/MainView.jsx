import React from 'react';
import Icon from '../../components/Icon/Icon';

const MainView = () => {
    return (
        <React.Fragment>
            <Icon name='globe' />
            &nbsp;
            Main View
        </React.Fragment>
    );
};

export default MainView;
