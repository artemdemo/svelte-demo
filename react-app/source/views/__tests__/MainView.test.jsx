import React from 'react';
import renderer from 'react-test-renderer';
import MainView from '../components/MainView';

jest.mock('../../components/Icon/Icon');

describe('<MainView>', () => {
    it('Empty render', () => {
        const tree = renderer.create(
            <MainView />,
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
