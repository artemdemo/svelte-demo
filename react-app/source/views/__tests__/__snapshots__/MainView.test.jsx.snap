// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainView> Empty render 1`] = `
Array [
  <div
    data-mock="Icon"
  >
    name: 
    globe
    className: 
    style: 
  </div>,
  "Â  Main View",
]
`;
